src/ViewEngine.ts

```diff
@@ -12,7 +12,8 @@ export abstract class ViewEngine {
     const joinPath = ctx.app.options.appRoot
       ? join(ctx.app.options.appRoot, this.path, file)
       : join(this.path, file);
-    const f = await readFileStr(joinPath);
+    const decoder = new TextDecoder('utf-8');
+    const f = decoder.decode(await Deno.readFile(joinPath));
```

Comment:
This can be replaced by this:
```suggestion
    const f = Deno.readTextFile(joinPath);
```